packages/examples/packages/name-lookup/src/index.ts

```diff
@@ -14,11 +13,11 @@ import { numberToHex } from '@metamask/utils';
 export const onNameLookup: OnNameLookupHandler = async (request) => {
   const { chainId, address, domain } = request;
 
+  // eslint-disable-next-line no-console
+  console.log(`Resolving for CAIP-2 chainId of: ${chainId}`);
+
   if (address) {
-    const shortAddress = `0x${address.substring(2, 5)}`;
-    const chainIdDecimal = parseInt(chainId.split(':')[1], 10);
-    const chainIdHex = numberToHex(chainIdDecimal);
-    const resolvedDomain = `example.domain - ${shortAddress} / ${chainIdHex}`;
+    const resolvedDomain = 'test.domain';
```

Comment:
I agree with this change, but wanna check with @matthewwalsh0 how much this is gonna break on the petnames E2E side. Any thoughts?

Reply:
I like that idea, what do you think @hmalik88 ?

Reply:
Sure, that works.